src/Twig/RuntimeLoader.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Bolt\Twig;
+
+use Pimple as Container;
```

Comment:
Thought you'd love that one … it was done partially with you in mind, and partially as it is simpler (IMHO) to see/change when we make the Pimple 3 switch in `master`